state-sync/storage-service/server/src/tests.rs

```diff
@@ -124,9 +142,9 @@ async fn test_get_storage_server_summary() {
                 highest_version,
             )),
             epoch_ending_ledger_infos: Some(CompleteDataRange::from_genesis(highest_epoch - 1)),
-            transactions: Some(CompleteDataRange::from_genesis(highest_version)),
-            transaction_outputs: Some(CompleteDataRange::from_genesis(highest_version)),
-            account_states: Some(CompleteDataRange::from_genesis(highest_version)),
+            transactions: Some(CompleteDataRange::new(10, highest_version).unwrap()),
+            transaction_outputs: Some(CompleteDataRange::new(20, highest_version).unwrap()),
+            account_states: Some(CompleteDataRange::new(51, highest_version).unwrap()),
```

Comment:
Are these numbers set in stone, just cause they all seem very arbitrary.  Magic numbers scare me.

Reply:
Nah, they're just random test numbers. But, they need to align with the mock storage implementation. Let me pull them into constants.